docs/release-notes/snapcraft-8-7.rst

```diff
@@ -0,0 +1,147 @@
+.. _release-8.7:
+
+Snapcraft 8.7 release notes
+=============================
+
+03 March 2025
+
+Learn about the new features, changes, and fixes introduced in Snapcraft 8.7.
+
+
+Requirements and compatibility
+------------------------------
+
+See :ref:`System requirements<system-requirements>` for information on the minimum
+hardware and installed software.
+
+
+What's new
+----------
+
+Snapcraft 8.7 brings the following features, integrations, and improvements.
+
+
+Bash completion
+~~~~~~~~~~~~~~~
+
+Previously, the completion file for the Snapcraft snap was outdated or incorrect
```

Comment:
Do we need to specify "snap"? "Snapcraft snap" tastes a bit unsavoury.

Reply:
Nope, dropped